locales/de/common.json

```diff
@@ -3,5 +3,7 @@
   "date": "Datum",
   "currentEvents": "Aktuelle Veranstaltungen",
   "thereAreCurrentlyNoEvents": "Es gibt aktuell keine Veranstaltungen.",
-  "AndMore": "Und noch weitere..."
+  "AndMore": "Und noch weitere...",
+  "creatingPdf": "Ihre PDF wird erstellt...",
+  "downloadPdfAt": "Sie können PDF hier herunterladen:"
```

Comment:
Also add these keys to translation Konstanten_div_Oktober2017-translated in drive

Reply:
done.